package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "flow": "flow --show-all-errors .",
+    "flow:all": "flow --show-all-errors .",
```

Comment:
why changing this? this will break the CI I guess

Reply:
fixed